m4/ax_lib_mapnik.m4

```diff
@@ -57,7 +58,7 @@ AC_DEFUN([AX_LIB_MAPNIK],
     )
 
     MAPNIK_CFLAGS=""
-    MAPNIK_LDFLAGS=""
+    MAPNIK_LIBS=""
```

Comment:
This variable was changed from `MAPNIK_LDFLAGS` to `MAPNIK_LIBS` in order to match the default variable names set up by `PKG_CHECK_MODULES`.

I.E.:
https://autotools.info/pkgconfig/pkg_check_modules.html#idm45459109695696